numpy/distutils/msvccompiler.py

```diff
@@ -0,0 +1,17 @@
+import os
+from distutils.msvccompiler import *
```

Comment:
See above about `*` imports. I think it would be clearer to use `distutils.MSVCCompiler`, why mess with an `_`.